netbox/core/tables/tasks.py

```diff
@@ -0,0 +1,73 @@
+import django_tables2 as tables
+from django_tables2.utils import A  # alias for Accessor
+from django.urls import reverse
+from django.utils.html import mark_safe
+from django.utils.translation import gettext_lazy as _
+
+from netbox.tables import NetBoxTable, columns
+from utilities.templatetags.helpers import annotated_date
+from ..models import Job
+
+
+class BackgroundTasksTable(tables.Table):
+    name = tables.LinkColumn("core:background_tasks_queues", args=[A("index")], verbose_name=_("Name"))
+    jobs = tables.LinkColumn("core:background_tasks_queues", args=[A("index")], verbose_name=_("Queued Jobs"))
+    oldest_job_timestamp = tables.Column(verbose_name=_("Oldest Queued Job"))
+    started_jobs = tables.Column(verbose_name=_("Active Jobs"))
```

Comment:
We need to be more careful about using "tasks" versus "jobs." This view is showing RQ tasks, and the column headers should reflect that. (IMO it's reasonable to omit the second word and just show e.g. "Active" or "Deferred.")